server.js

```diff
@@ -1,29 +1,84 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-import mongoose from 'mongoose'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
-mongoose.Promise = Promise
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
+import express from 'express'; 
+import bodyParser from 'body-parser';
+import cors from 'cors';
+import mongoose from 'mongoose';
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+mongoose.Promise = Promise;
+
+// Defines the port the app will run on. 
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+
+// Thought model
+ const Thought = mongoose.model('Thought', {
+   message: { 
+     type: String,
+     required: true,
+     minLength: 5,
+     maxLength: 140
+   },
+
+   hearts: { 
+     type: Number,
+     default: 0
+   },
+
+   createdAt: { 
+     type: Date, 
+     default: Date.now
+   }
+ });
 
-// Start defining your routes here
+// ROUTES
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send('Hello and Welcome to the Happy thoughts API, where feeling blue is not allowed! :)');
```

Comment:
I'm blue
Da ba dee da ba di...

Sorry I couldn't help myself 😸

Reply:
da ba di da bi daaaa
😂 loved it!